styleguide/derek/view-templates/leadership/_leadership.scss

```diff
@@ -0,0 +1,49 @@
+.leadership-individual {
+  @include make-sm-column(3);
+  @include make-xs-column(6);
+  margin: 20px auto;
+}
+
+.leadership-image {
+  @include center-block;
+}
+
+.leadership-attrWrapper {
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.leadership-name {
+  margin-bottom: 0;
+}
+
+.leadership-title {
+  margin: 0;
+}
+
+.leadership-linkWrapper {
+  margin-top: 20px;
+}
+
+.leadership-link {
+  @include standard-link;
+}
+
+@media only screen and (min-width: $screen-xs-max) {
+  .leadership-individual {
+    &.center-block {
+      float: none;
+    }
+  }
+}
+
+@media only screen and (max-width: $screen-sm-max) {
```

Comment:
Can we change this to 
```
@media only screen and (min-width: $screen-md-min) {
```

And then setup then set the below behavior as the default, and then override it for `md+` sizes. Eventually we're gonna start addressing media queries in RSWEB-8664, but the fewer queries we have to override at that point the better.